requirements.txt

```diff
@@ -1,33 +1,14 @@
-astroplan
-astropy >= 3.0.0
-ccdproc
-codecov
-coveralls
-dateparser
+# These are packages that are NOT available via our
+# choice of conda repositories; there are less well
+# known repositories that do offer them, but that is
+# more of a security risk.
+
+# Python wrapper for ffmpeg.
 ffmpy
```

Comment:
We could probably lose the option. It is just a thin wrapper around ffmpeg. Since we don't have many varying options it would be trivial to write our own script.